### PR TITLE
Upgrade to Elasticsearch v2.3.3

### LINKED ISSUE #38: Please update to 2.3.3
I am using another plugin which only supports 2.3.3 version of elasticsearch, it would be great if elasticsearch-cloud-kubernetes plugin is updated to support 2.3.3 version of elasticsearch.

### PR DESCRIPTION
**Pull request by vaijab**
Fixes #38

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.fabric8</groupId>
   <artifactId>elasticsearch-cloud-kubernetes</artifactId>
-  <version>2.3.3-SNAPSHOT</version>
+  <version>2.3.3</version>
```
Comment (jimmidyson) [inline pom.xml]: This should stay a SNAPSHOT version until release please.

**Comment by vaijab** [comment]
@jimmidyson sorry, I was not aware of that. Updated, please check again.

**Comment by jimmidyson** [comment]
ok to test

**Comment by jimmidyson** [comment]
LGTM - just waiting for green to merge.

**Comment by jimmidyson** [comment]
[merge]

**Comment by jimmidyson** [comment]
Release should be available in ~20 minutes from Maven central.

**Comment by vaijab** [comment]
@jimmidyson got it already. Thanks for such a quick turnaround.

**Comment by jimmidyson** [comment]
Thank you for the PR!

**Pull request merged**